cms/templates/js/self-paced-due-date-editor.underscore

```diff
@@ -15,6 +15,10 @@
     </li>
 </ul>
 
+<div id="projected" class="message">
+    <%- gettext('If a learner starts on ') %> <span id="startDate"></span><%- gettext(', this subsection will be due on ') %><span id="projectedDueIn"></span>
```

Comment:
Ideally you can do some sort of formatting here to keep all of the gettext call together. Each string is translated on its own so it's important to keep the full context in the string so translators don't just see "If a learner starts on" and ", this subsection will be due on"
For example:
`gettext('If a learner starts on {start_date}, this subsection will be due on {projected_date}).format(start_date='<span id="startDate"></span>', projected_date='<span id="projectedDueIn"></span>')`

Although hmm. I'm not sure that would allow your dom manipulation. Would be good to play around this though and see what we can make work.